### PR TITLE
integration-tests: Run testSubAndInAndFail with "setenforce 0"

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
The main idea here is to run the whole test and not have it fail in the middle.  We will skip with a known issue in any case, but running the whole test is better than running only half.

The "systemctl start insights-client" command will only sometimes fail, depending on how fast and where the insights-client program itself fails.  But we also want it to succeed eventually, and we want to see all denials, and not just the first that makes it fail.

File: integration-tests/check-subscriptions [hunk 1]
```diff
@@ -402,6 +402,14 @@ class TestSubscriptions(SubscriptionsCase):
         m = self.machine
         b = self.browser
 
+        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=2062136
+        #
+        # We rely on insights-client.service to be working, let's not
+        # get distracted by SELinux.  Denials will still be logged and
+        # tracked as known issues even when not enforcing.
+        #
+        m.execute("setenforce 0")
```
Comment (jirihnidek) [inline integration-tests/check-subscriptions:411]: Is this only temporary workaround? Will it be possible to revert, when https://bugzilla.redhat.com/show_bug.cgi?id=2062136 is properly fixed?

**Comment by mvollmer** [comment]
> Hm shouldn't the same workaround apply also to `testInsights`?

SELinux only interferes with insights-client when the latter is run via systemd.  `testInsights` doesn't need those runs to succeed, so `setenforce 0` wouldn't make a difference there as far as the test run is concerned.  We will probably get a different set of "unexpected journal messages", though, but I don't think we care about them too much.

But there also wouldn't be anything wrong with running `testInsights` with `setenforce 0`... do you think we should do it?

**Comment by ptoscano** [comment]
> SELinux only interferes with insights-client when the latter is run via systemd.

great :unamused: ...

I see in the logs for this test:
> > warning: Parse error /var/lib/insights/host-details.json SyntaxError: Unexpected end of JSON input

do you have an example of that file? is it empty perhaps?

**Pull request merged**